src/onlyspans.polyleads.ui/tsconfig.json

```
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
```

если получится, то лучше так

```suggestion
    "allowJs": false,
```